targets/TARGET_NXP/TARGET_LPC176X/TARGET_MBED_LPC1768/PinNames.h

```diff
@@ -106,11 +106,17 @@ typedef enum {
     A4 = P1_30,
     A5 = P1_31,
 
-    I2C_SCL = D15,
-    I2C_SDA = D14,
-
     // Not connected
-    NC = (int)0xFFFFFFFF
+    NC = (int)0xFFFFFFFF,
```

Comment:
The thing I was attempting to do here was use NC to indicate I2C0 was not connected on the LPC1768 and allow any error-checking code to kick in. In order to do that, it needs to be ahead of these lines:

I2C_SCL0 = NC,
I2C_SDA0 = NC,

Alternatively, I could just omit these lines and leave NC at the bottom of the file.